src/components/FeedbackModal/FeedbackModal.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  Heading,
+} from "@chakra-ui/react"
+import { ModalCloseButton } from "@opengovsg/design-system-react"
+
+const FeedbackForm = () => {
+  return (
+    <>
+      <div
+        style={{
+          fontFamily: "sans-serif",
+          fontSize: "15px",
+          color: "#000",
+          opacity: "0.9",
+          paddingTop: "5px",
+          paddingBottom: "8px",
+        }}
+      >
+        If the form below is not loaded, you can also fill it in{" "}
+        <a href="https://form.gov.sg/64b7a11823e54700118bad90">here</a>.
+      </div>
+
+      <iframe
+        title="FormSG Feedback form for Isomer"
+        id="iframe"
+        src="https://form.gov.sg/64b7a11823e54700118bad90"
+        width="100%"
+        height="650px"
```

Comment:
curious, can this be 100% (or similar) to fill the modal height instead of scrolling? Since it is a short form, can be embedded to fullheight right? might look nicer to user

but if not, am fine with this also

Reply:
i actually didn't want to edit this because this is as-is by forms. i could try but fwiw, i'm rolling out a UI rehash (PM asked) so it might not be of value

Reply:
Okay, let's proceed as it is now